recipe/meta.yaml

```diff
@@ -28,12 +28,11 @@ test:
     - nose-cov
     - chai
     - simplejson
-  commands:
-    - nosetests "{{ environ['SRC_DIR'] }}"
```

Comment:
Latest `conda-build` is not happy with the `SRC_DIR` in test.

@msarahan can you point me to the docs of the new way to do this? I could not find it.